src/components/graphs/linegraph/LineGraph.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Line } from 'react-chartjs-2';
+import { options } from '../assets';
+
+const LineGraph = ({ data, months }) => {
+  const [inCategories, setInCategories] = useState({});
+  const [labels, setLabels] = useState([]);
+  const [incidentID, setIncidentID] = useState('all');
+  const [label, setLabel] = useState('All');
+  const [lineData, setLineData] = useState([]);
+
+  useEffect(() => {
+    setLabels(months);
+  }, [data, months]);
+
+  useEffect(() => {
+    if (
+      !(Object.keys(data).length === 0 && inCategories.constructor === Object)
+    ) {
+      let d = [];
```

Comment:
Please always use declarative variable names. It's not clear what 'd' is or what it represents. Please update ASAP. Let me know if you aren't familiar with the renaming shortcut (not find/replace).

Reply:
I've renamed them